spec/certificate_request_spec.rb

```diff
@@ -130,10 +130,10 @@
     request = Acme::Client::CertificateRequest.new(names: %w(example.org www.example.org), private_key: test_key)
 
     extension = request.csr.attributes.find { |attribute|
-      attribute.value.first.first.value.first.value == 'subjectAltName'
```

Comment:
Such a weird API change. Do you happen to know what lead to that change? I couldn't find anything in the openssl change log / repo.